modules/contact-form/grunion-contact-form.php

```diff
@@ -848,11 +848,13 @@ public function get_post_content_for_csv_export( $post_id ) {
 	 * @param int $post_id Id of the post to fetch meta data for.
 	 *
 	 * @return mixed
-	 *
-	 * @codeCoverageIgnore - No need to be covered.
 	 */
 	public function get_post_meta_for_csv_export( $post_id ) {
-		return get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md                  = get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md['feedback_date'] = get_the_date( 'Y-m-d H:i:sT', $post_id );
+		$content_fields      = self::parse_fields_from_content( $post_id );
+		$md['feedback_ip']   = $content_fields['_feedback_ip'];
```

Comment:
```suggestion
		$md['feedback_ip']   = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
```

It's possible to edit the content of the feedback and remove the IP, so we should check before we assume it is present.

I'm also suggesting returning 0 as an unknown value.

Some day when PHP 7 is the minimum, would be a good use of `??` too. Alas.

Reply:
did not know ip could b removed, thanx